py/py3/ocutils.py

```diff
@@ -313,15 +309,15 @@ def process_level(self, level, old_models, clear_cache_flag):
                 len(best_models) < self._search_width
             ):  # or key[0] == last_key[0]:      # comparing keys allows us to select more than <width> models,
                 if True not in [
-                    n.isEquivalentTo(candidate) for n in best_models
+                    n.is_is_equivalent_to(candidate) for n in best_models
```

Comment:
is this name a bit redundant? is_is_equivalent_to()

Reply:
This method has been removed in the new commit